ui/pages/confirmations/components/confirm/info/approve/edit-spending-cap-modal/edit-spending-cap-modal.tsx

```diff
@@ -171,6 +173,15 @@ export const EditSpendingCapModal = ({
               {t('editSpendingCapError', [decimals])}
             </Text>
           )}
+          {showSpecialCharacterError && (
+            <Text
```

Comment:
Minor, is it worth a local `Error` component to avoid the duplication with the above decimal error?